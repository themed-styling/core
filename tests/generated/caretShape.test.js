import 'jest-styled-components'

import { caretShape } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(caretShape, 'caretShape')
