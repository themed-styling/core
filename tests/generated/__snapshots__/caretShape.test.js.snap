// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`caretShape renders 0 1`] = `
Array [
  .c0 {
  caret-shape: 0;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: 0;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: 0;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`caretShape renders function 1`] = `
Array [
  .c0 {
  caret-shape: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: has;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: has;
}

<div
    className="c0"
  />,
]
`;

exports[`caretShape renders invalid theme path 1`] = `
Array [
  .c0 {
  caret-shape: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`caretShape renders negative number 1`] = `
Array [
  .c0 {
  caret-shape: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`caretShape renders object [breakpoints + standard] 1`] = `
Array [
  .c0 {
  caret-shape: July;
}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    caret-shape: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: July;
}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    caret-shape: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    caret-shape: June;
  }
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    caret-shape: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`caretShape renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    caret-shape: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    caret-shape: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    caret-shape: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    caret-shape: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    caret-shape: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    caret-shape: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    caret-shape: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`caretShape renders object [mediaQueries + standart] 1`] = `
Array [
  .c0 {
  caret-shape: July;
}

@media May {
  .c0 {
    caret-shape: August;
  }
}

@media April {
  .c0 {
    caret-shape: September;
  }
}

@media January {
  .c0 {
    caret-shape: November;
  }
}

@media April {
  .c0 {
    caret-shape: August;
  }
}

@media June {
  .c0 {
    caret-shape: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: July;
}

@media May {
  .c0 {
    caret-shape: August;
  }
}

@media April {
  .c0 {
    caret-shape: September;
  }
}

@media January {
  .c0 {
    caret-shape: November;
  }
}

@media April {
  .c0 {
    caret-shape: August;
  }
}

@media June {
  .c0 {
    caret-shape: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    caret-shape: November;
  }
}

@media April {
  .c0 {
    caret-shape: January;
  }
}

@media January {
  .c0 {
    caret-shape: December;
  }
}

@media April {
  .c0 {
    caret-shape: April;
  }
}

@media June {
  .c0 {
    caret-shape: May;
  }
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    caret-shape: November;
  }
}

@media April {
  .c0 {
    caret-shape: January;
  }
}

@media January {
  .c0 {
    caret-shape: December;
  }
}

@media April {
  .c0 {
    caret-shape: April;
  }
}

@media June {
  .c0 {
    caret-shape: May;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`caretShape renders object [mediaQueries] 1`] = `
Array [
  @media May {
  .c0 {
    caret-shape: July;
  }
}

@media April {
  .c0 {
    caret-shape: August;
  }
}

@media January {
  .c0 {
    caret-shape: September;
  }
}

@media April {
  .c0 {
    caret-shape: November;
  }
}

@media June {
  .c0 {
    caret-shape: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    caret-shape: July;
  }
}

@media April {
  .c0 {
    caret-shape: August;
  }
}

@media January {
  .c0 {
    caret-shape: September;
  }
}

@media April {
  .c0 {
    caret-shape: November;
  }
}

@media June {
  .c0 {
    caret-shape: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    caret-shape: January;
  }
}

@media April {
  .c0 {
    caret-shape: December;
  }
}

@media January {
  .c0 {
    caret-shape: April;
  }
}

@media April {
  .c0 {
    caret-shape: May;
  }
}

@media June {
  .c0 {
    caret-shape: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    caret-shape: January;
  }
}

@media April {
  .c0 {
    caret-shape: December;
  }
}

@media January {
  .c0 {
    caret-shape: April;
  }
}

@media April {
  .c0 {
    caret-shape: May;
  }
}

@media June {
  .c0 {
    caret-shape: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`caretShape renders postitive number 1`] = `
Array [
  .c0 {
  caret-shape: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`caretShape renders string 1`] = `
Array [
  .c0 {
  caret-shape: July;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: July;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: June;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: June;
}

<div
    className="c0"
  />,
]
`;

exports[`caretShape renders undefined 1`] = `
Array [
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
]
`;

exports[`caretShape renders valid theme path 1`] = `
Array [
  .c0 {
  caret-shape: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  caret-shape: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
